### ISSUE TITLE
Remove doctype from XML preamble

### ISSUE THREAD

**Issue by rjsparks** [ietf-tools/bibxml-service#252]
If this is already being tracked elsewhere, point me to it please.

Currently the bibxml being produced includes a doctype declaration pointing to rfc2629.dtd. 

This is problematic for many reasons that are out of scope for this project to deal with. 

For now, the references should not include that declaration at all - using documents will have an appropriate declaration at their top level.

Please remove it from the generation of the bibxml at this service.

See for example:
```
view-source:https://bib.ietf.org/public/rfc/bibxml3/reference.I-D.ietf-sipcore-multiple-reasons.xml
```

```
<?xml version='1.0' encoding='utf-8'?>
<!DOCTYPE reference SYSTEM "rfc2629.dtd">
<reference anchor="I-D.ietf-sipcore-multiple-reasons" target="https://www.ietf.org/archive/id/draft-ietf-sipcore-multiple-reasons-00.txt">
  <front>
    <title>Multiple SIP Reason Header Field Values</title>
    <author fullname="Robert Sparks"/>
    <date day="25" month="July" year="2022"/>
    <abstract>
      <t>The SIP Reason Header Field as defined in RFC 3326 allows only one Reason value per protocol value. Practice shows it is useful to allow multiple values with the same protocol value. This update to RFC 3326 allows multiple values for an indicated registered protocol when that protocol defines what the presence of multiple values means.</t>
    </abstract>
  </front>
  <seriesInfo name="Internet-Draft" value="draft-ietf-sipcore-multiple-reasons-00"/>
</reference>
```

**Comment by strogonoff** [comment]
> If this is already being tracked elsewhere, point me to it please.

This is definitely not tracked anywhere, since doctype was added just last week (based on https://github.com/ietf-ribose/bibxml-service/issues/234). It can be removed just as easily, of course.

**Comment by rjsparks** [comment]
Yes please remove it.

**Comment by kesara** [comment]
Fixed in `v2022.8.8_1`
